### ISSUE TITLE
Snowflake (v3.11) table SQL requires filter by type (and database in URL)

### ISSUE THREAD

**Issue by brianrepko** [schemaspy/schemaspy#504]
Snowflake just upgraded from 3.10 to 3.11 and I'm using the new PR for snowflake support.

The SQL for getting tables and views requires
1. The database to be on the URL so that the context for `information_schema` is set
2. The table SQL needs to filter by table_type = 'BASE TABLE' as views are now in the tables view (otherwise you get duplicates for views - they show up as both tables and views)

I think some of this might be changes to Snowflake (v3.11 and updated JDBC driver) but might also be mistakes in my previous PR. I'm now running the actual schemaspy code vs my own.

PR to fix this will come next.

**Comment by npetzall** [comment]
I'm guessing that there is no way of running 3.10, since it's hosted solution?

I asking since for most databaseTypes we version the configuration. Which wouldn't be needed for snowflake since there is no possibility of running an older version?

**Comment by brianrepko** [comment]
That is correct as far as I know though I could imagine that a Snowflake warehouse (the compute component) might support some limited set of versions. Let me check on that

**Comment by brianrepko** [comment]
The is the version of the host and the version of the JDBC driver. Snowflake owns the host and recommends you use the latest client.

**Comment by npetzall** [comment]
So what is your take on this. I sort assume that the database layout/structure is not related to driver.
So there is no reason to version the snowflake databasetype. It should always be "latest".

**Comment by brianrepko** [comment]
My opinion is that it should always be "latest".

**Comment by npetzall** [comment]
If your satisfied with the PR I'll merge it.

**Comment by brianrepko** [comment]
Yes - we have been actively using this
